roles/oneview_server_profile_facts/README.md

```diff
@@ -0,0 +1,37 @@
+## oneview_server_profile_facts
+Manage OneView Server Profile resources.
+
+#### Synopsis
+ Provides an interface to manage Server Profile resources. Can create, update, and delete.
```

Comment:
synopsis should say about facts. Please change.
``` Retrieve facts about the Server Profile from OneView.
```

Reply:
done